alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -1016,6 +1016,86 @@ public static function resetEntryStatistics(entry $entry)
  		$entry->setRank(0);
  		$entry->setTotalRank(0);
 	}
+
+	public static function copyEntryData(entry $entry, entry $targetEntry)
+	{
+		// for any type that does not require assets:
+		$shouldCopyDataForNonClip = true;
+		if ($entry->getType() == entryType::MEDIA_CLIP)
+			$shouldCopyDataForNonClip = false;
+		if ($entry->getType() == entryType::PLAYLIST)
+			$shouldCopyDataForNonClip = false;
+
+		$shouldCopyDataForClip = false;
+		// only images get their data copied
+		if($entry->getType() == entryType::MEDIA_CLIP)
+		{
+			if($entry->getMediaType() != entry::ENTRY_MEDIA_TYPE_VIDEO &&
```

Comment:
@tan-tan-kanarek note this code is not new, I only moved it from a different function. Are we sure image is the only possibility here?